moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -316,10 +317,6 @@ def set_random_target(self):
         """ Set a random joint configuration target """
         self._g.set_random_target()
 
-    def get_named_targets(self):
```

Comment:
I don't see a reason to remove this. I think it's by mistake.

Reply:
Correct, I had added it back (in a new commit, the MR should be squashed anyways